lib/core/src/lib/notifications/components/notification-history.component.ts

```diff
@@ -53,17 +52,19 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
     onDestroy$ = new Subject<boolean>();
     notifications: NotificationModel[] = [];
     paginatedNotifications: NotificationModel[] = [];
+    unreadNotifications: NotificationModel[] = [];
     pagination: PaginationModel;
+    badgeHidden: boolean;
```

Comment:
this is not needed, already covered by `unreadNotifications` checks

Reply:
removed